ee/tabby-webserver/src/service/auth.rs

```diff
@@ -669,7 +524,7 @@ mod tests {
             service
                 .token_auth(ADMIN_EMAIL.to_owned(), "12345678".to_owned())
                 .await,
-            Err(TokenAuthError::UserNotFound)
+            Err(_)
```

Comment:
nit: maybe prefer `assert!(operation.is_err())`?

Reply:
ack - leaving it as is just for cleaner diff.